.github/workflows/stage-deploy.yml

```diff
@@ -20,25 +28,26 @@ jobs:
             - uses: actions/checkout@v3
               with:
                   repository: mdn/bob
-                  ref: stage
+                  ref: main
                   path: bob
 
             - name: Setup Node.js environment
               uses: actions/setup-node@v3
               with:
                   node-version: '16'
-                  cache: yarn
+                  cache: npm
+                  cache-dependency-path: bob/package-lock.json
 
-            - name: Install all yarn packages
-              run: yarn --production --frozen-lockfile
+            - name: Install dependencies of bob
+              run: npm ci
               working-directory: bob
 
             - name: Build all
               # TODO: bob uses relative paths in bundles. For now we just symlink.
               run:  |
-                  ln -s ../bob/node_modules .
-                  node ../bob/lib/mdn-bob.js
-              working-directory: examples
+                  rm -rf ./docs
+                  npm run build ../examples
```

Comment:
This doesn't actually build the examples in `../examples`, just bob's own (test) examples.